API/projectClass/projectClass.m

```diff
@@ -188,18 +190,7 @@
             invalidTypeMessage = sprintf('Geometry must be a geometryOptions enum or one of the following strings (%s)', ...
                                          strjoin(geometryOptions.values(), ', '));
             obj.geometry = validateOption(geometry, 'geometryOptions', invalidTypeMessage).value;
-        end
-        
-        function obj = setModelType(obj, modelType)
```

Comment:
The "setModelType" routine has been used widely in the past, and is used throughout the examples, so we'll need to discuss its removal carefully. The case for doing so is that the model type sets what is allowed elsewhere, and changing it on the fly could lead to many unintended consequences. For example, a set of contrasts could be defined using layers, with the model type then changed to custom where the layers details are removed and the contrasts have dangling references.